app/src/main/java/ru/chaichuk/hwapp/data/Movie.kt

```diff
@@ -0,0 +1,65 @@
+package ru.chaichuk.hwapp.data
+
+import android.os.Parcel
+import android.os.Parcelable
+
+data class Movie(
+    val id: Int,
+    val title: String,
+    val overview: String,
+    val poster: String,
+    val backdrop: String,
+    val ratings: Float,
+    val numberOfRatings: Int,
+    val minimumAge: Int,
+    val runtime: Int,
+    val genres: List<Genre>,
+    val actors: List<Actor>,
+
+    val like: Boolean
+) : Parcelable {
+    constructor(parcel: Parcel) : this(
```

Comment:
**it's optional**
instead of get extend from parcel you can use [kotlin parselize ](https://developer.android.com/kotlin/parcelize)

Reply:
Moved to kotlin's parcelize generator